specification/cognitiveservices/data-plane/SpeechServices/stable/v2.0/Speech.json

```diff
@@ -12239,37 +12188,37 @@
     "title": "IReadOnlyDictionary`2",
     "type": "object",
     "properties": {
-      "None": {
+      "none": {
```

Comment:
does the casing update reflect the service? Overall, we need to make sure the swagger reflects the service, so if updates made to the swagger due to issues reported by tools do not reflect the service, and the service can't be changed for this api version, you can ask for exceptions for the issues.